android/java/org/chromium/chrome/browser/app/domain/KeyringModel.java

```diff
@@ -121,26 +125,20 @@ private void update(int coinType) {
             mKeyringService.getKeyringInfo(getSelectedCoinKeyringId(coinType),
                     keyringInfo -> { _mSelectedCoinKeyringInfoLiveData.postValue(keyringInfo); });
             mKeyringService.getKeyringsInfo(mSharedData.getEnabledKeyrings(), keyringInfos -> {
-                List<AccountInfo> accountInfos =
+                final List<AccountInfo> accountInfos =
```

Comment:
I was tempted to modify `accountInfos` -> `accountInfoList` as it doesn't sound right. 
I refrained from doing it as there are 36 occurrences of this term, and it would make reviewing the `KeyringModel` class a nightmare. It's not a big deal in my opinion, so we can leave it as it is for now. 
If you want me to rename anyway, let me know.